ci_framework/roles/devscripts/vars/main.yml

```diff
@@ -39,7 +39,8 @@ cifmw_devscripts_config_defaults:
   base_domain: "openstack.lab"
   ntp_servers: "clock.corp.redhat.com"
   ip_stack: "v4"
-  network_type: "OpenShiftSDN"
+  network_type: "OVNKubernetes"
+  ovn_local_gateway_mode: "false"
```

Comment:
query: shouldn't this be `true` by default.

Reply:
I kept same as Openshift default, false i.e shared mode and it works for most of the use cases. and enabled only on the scenario where we manage libvirt networks. My idea was to enable it only when really needed else not touch it.

Reply:
Thanks @karelyatin for the explanation.